pydatastructs/.gitignore

```diff
@@ -0,0 +1,5 @@
+**/venv/
+# Editor Files 
+**/.vscode/
+# Scripting Files
+**/run.py
```

Comment:
This file should be deleted. There is already a `.gitignore` available under the root directory.

Reply:
Oh okay.